applications/GeoMechanicsApplication/tests/cpp_tests/custom_elements/test_integration_schemes.cpp

```diff
@@ -114,8 +152,10 @@ KRATOS_TEST_CASE_IN_SUITE(PointsOfAllSupportedLobattoSchemesMustBeInRangeAndIncl
     for (auto number : SupportedNumbersOfPointsForLobattoIntegration()) {
         const auto scheme = MakeLobattoIntegrationScheme(number);
 
-        ExpectLocalCoordinatesAreInRange(scheme->GetIntegrationPoints());
-        ExpectLocalCoordinatesIncludeRangeBounds(scheme->GetIntegrationPoints());
+        ExpectLocalCoordinatesAreInRange(scheme->GetIntegrationPoints(), -1.0, 1.0, std::size_t{0});
+        ExpectLocalCoordinatesAreZero(scheme->GetIntegrationPoints(), std::size_t{1});
+        ExpectLocalCoordinatesAreZero(scheme->GetIntegrationPoints(), std::size_t{2});
+        ExpectLocalCoordinatesIncludeRangeBounds(scheme->GetIntegrationPoints(), -1.0, 1.0, std::size_t{0});
```

Comment:
Perhaps we can reorder these checks, such that we first check $`\xi`$, then $`\eta`$, and finally $`\zeta`$? That is:
```suggestion
        ExpectLocalCoordinatesAreInRange(scheme->GetIntegrationPoints(), -1.0, 1.0, std::size_t{0});
        ExpectLocalCoordinatesIncludeRangeBounds(scheme->GetIntegrationPoints(), -1.0, 1.0, std::size_t{0});
        ExpectLocalCoordinatesAreZero(scheme->GetIntegrationPoints(), std::size_t{1});
        ExpectLocalCoordinatesAreZero(scheme->GetIntegrationPoints(), std::size_t{2});
```

Reply:
done